src/main/java/step3/Count.java

```diff
@@ -3,29 +3,33 @@
 import java.util.Objects;
 
 public class Count {
-    private int ZERO = 0;
+    private final static int MIN = 1;
```

Comment:
자바 관례상 키워드 우선순위는 static이 final보다 높습니다. 해당 링크를 참고해주세요 ㅎ ([링크](https://johngrib.github.io/wiki/java-modifier-order-with-jls-suggestion/))

```suggestion
    private static final int MIN = 1;
```